### DISCUSSION TITLE
I do not konw how to config extendMarkdown

### DISCUSSION THREAD

**Discussion by shirayner** [vuepress-theme-hope/vuepress-theme-hope#2775] [Q&A] [answered]
when i config extendMarkdown, there are some editor error:

![image](https://user-images.githubusercontent.com/11574340/218384361-caf4f377-e440-46f6-a6b3-bb58537d8875.png)

![image](https://user-images.githubusercontent.com/11574340/218384406-365fe36d-43fb-4d21-94a7-df217e30b877.png)

reference doc:
- https://v1.vuepress.vuejs.org/zh/plugin/option-api.html#extendmarkdown

**Comment by Mister-Hope** [comment, marked as answer]
https://github.com/vuepress-theme-hope/vuepress-theme-hope#details You will get open source author annoyed if you do not read docs.
